pkg/controllers/user/helm/controller.go

```diff
@@ -116,7 +116,7 @@ func (l *Lifecycle) Updated(obj *v3.App) (runtime.Object, error) {
 	// always refresh app to avoid updating app twice
 	_, projectName := ref.Parse(obj.Spec.ProjectName)
 	var err error
```

Comment:
Can you add a comment here explaining why this can't be a lister, so someone doesn't switch in the future trying to make performance improvement.